sdcm/cluster_docker.py

```diff
@@ -52,12 +52,18 @@ def node_container_run_args(self, seed_ip):
         volumes = {
             '/var/run/docker.sock': {"bind": '/var/run/docker.sock', "mode": "rw"},
         }
+
+        # TODO: add parsing of cpuset option from append_scylla_args parameter
```

Comment:
Why the TODO, looks like you are doing just that

Reply:
> Why the TODO, looks like you are doing just that

This is about cpuset parameter that can be passed to scylla. In this case the variability of values for the parameter require smore complex logic to parse it. Like here https://github.com/scylladb/scylla-cluster-tests/blob/4121b79f03e891dc1503f536fdb2866c4f1e37e8/sdcm/cluster.py#L549

Reply:
I don't think we need to support it, it means we'll want to do CPU pinning, and that type of tuning it not in the scope of what we need from the docker backend